otlp/logs.go

```diff
@@ -84,7 +84,7 @@ func TranslateLogsRequest(ctx context.Context, request *collectorLogs.ExportLogs
 				}
 
 				// get sample rate after resource and scope attributes have been added
-				sampleRate := getSampleRate(attrs)
+				sampleRate := getSampleRate(attrs, "")
```

Comment:
I'm not entirely sure if this is even applicable to logs. Is it?

Reply:
Logs don't pass in their associated span's trace state so we're just passing in "". It's currently using the top level func, but could be made to just use the Honeycomb sample rate one.